edx_proctoring/tests/test_views.py

```diff
@@ -502,6 +502,36 @@ def test_attempt_readback(self):
         self.assertIsNotNone(response_data['started_at'])
         self.assertIsNone(response_data['completed_at'])
 
+    def test_attempt_ready_to_start(self):
+        """
+        Test to get an attempt with ready_to_start status
+        and will return the response_data with time time_remaining_seconds to 0
+        """
+        # Create an exam.
+        proctored_exam = ProctoredExam.objects.create(
+            course_id='a/b/c',
+            content_id='test_content',
+            exam_name='Test Exam',
+            external_id='123aXqe3',
+            time_limit_mins=90
+        )
+        attempt = ProctoredExamStudentAttempt.create_exam_attempt(
+            proctored_exam.id, self.user.id, 'test_user', 1,
+            'test_attempt_code', True, False, 'test_external_id',
+            status=ProctoredExamStudentAttemptStatus.ready_to_start
+        )
```

Comment:
Create an attempt, then set the status manually to "ready_to_start" and then save().